packages/frontend/src/pages/login.tsx

```diff
@@ -134,6 +135,14 @@ const Login: FC = () => {
                             data-cy="login-button"
                         />
                     </Form>
+                    {health.data?.auth.google.loginPath && (
```

Comment:
Shouldn't this be checking `health.data?.auth.google.oauth2ClientId` ? `loginPath` is always defined so this is always true